src/message.rs

```diff
@@ -145,6 +148,25 @@ macro_rules! message_args_impl {
                 })
             }
 
+            #[cfg(all(feature="gnustep_runtime",
+                      not(any(target_arch = "arm",
+                              target_arch = "x86",
+                              target_arch = "x86_64"))))]
+            unsafe fn send<T, R>(self, obj: *mut T, sel: Sel) -> R
+                   where T: Message, R: Any {
+                    let mut receiver = obj as *mut Object;
+                    let nil: *mut Object = ::std::ptr::null_mut();
+                    let ref slot = *runtime::objc_msg_lookup_sender(&mut receiver as *mut *mut Object, sel, nil);
+                    let imp_fn = slot.method;
+                    let imp_fn: unsafe extern fn(*mut Object, Sel $(, $t)*) -> R =
+                        mem::transmute(imp_fn);
+                    let ($($a,)*) = self;
+                    objc_try!({
+                      imp_fn(receiver as *mut Object, sel $(, $a)*)
```

Comment:
Does calling this function on a slot handle unrecognized selectors properly? The reason this crate is using `objc_msgSend` instead of looking up function pointers manually is because `objc_msgSend` will call `forwardInvocation:` and `doesNotRecognizeSelector:` when invoking the function manually won't.